src/libmongoc/src/mongoc/mongoc-collection.c

```diff
@@ -1952,12 +1965,18 @@ mongoc_collection_insert_many (mongoc_collection_t *collection,
       return false;
    }
 
+   if (insert_many_opts.crud.comment.value_type != BSON_TYPE_EOD) {
+      bson_append_value (
+         &cmd_opts, "comment", 7, &insert_many_opts.crud.comment);
+   }
+
+   if (!bson_empty (&insert_many_opts.extra)) {
+      bson_concat (&cmd_opts, &insert_many_opts.extra);
```

Comment:
I realize that in `mongoc_collection_insert_one` I did this backwards and appended `comment` before concatenating `extra`. Does that matter at all? I assume `extra` will never include any known options so there's no concern with a possible key conflict here (were that to happen, I expect the server would just use the first instance of `comment` in the command).

Happy to swap the order here if we'd like to be consistent though.